src/features/areas/components/PlaceCard.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  TextField,
+} from '@mui/material';
+import { FC, useState } from 'react';
+import { LatLng } from 'leaflet';
+import { makeStyles } from '@mui/styles';
+
+import messageIds from '../l10n/messageIds';
+import useCreatePlace from '../hooks/useCreatePlace';
+import { Msg, useMessages } from 'core/i18n';
+
+export const useStyles = makeStyles(() => ({
+  card: {
+    bottom: 15,
+    display: 'flex',
+    gap: 8,
+    justifyContent: 'center',
+    padding: 8,
+    position: 'absolute',
+    width: '100%',
+    zIndex: 1000,
+  },
+}));
+
+type AddPlaceDialogProps = {
+  onClose: () => void;
+  orgId: number;
+  point: LatLng | null;
+};
+
+export const PlaceCard: FC<AddPlaceDialogProps> = ({
+  onClose,
+  orgId,
+  point,
+}) => {
+  const classes = useStyles();
+  const messages = useMessages(messageIds);
+  const createPlace = useCreatePlace(orgId);
+  const [title, setTitle] = useState<string>('');
+  const [type, setType] = useState<string>('');
+
+  const handleChange = (event: SelectChangeEvent) => {
+    setType(event.target.value);
+  };
+
+  const placeholderText = () => {
+    if (type === 'address') {
+      return messages.placeCard.placeholderAddress();
+    } else if (type === 'misc') {
+      return messages.placeCard.placeholderMisc();
+    }
```

Comment:
I don't think this should be handled separately. The `placeholderTitle` works fine for misc places IMO.